core/src/main/resources/zeneventserver.conf

```diff
@@ -250,11 +250,12 @@ zep.redis.port=6379
 #zep.lucene.archive.honor.deletes=true
 
 # Zenoss Cloud configuration, disabled by default
-zep.zing.enabled=true
-zep.zing.use_emulator=true
-zep.zing.tenant=paco
-zep.zing.source=cz1
-zep.zing.project=zenoss-zing
-zep.zing.topic=events-in
-zep.zing.emulator_url=paco.zenoss.eng:8085
+zep.zing.enabled=false
+#zep.zing.use_emulator=true
```

Comment:
Why disable by default? Shouldn't it be enabled for CZ?  Or are you accounting for the 6.x case? Also, why comment out everything?

Reply:
accounting for 6.x. The config will be injected by CC. It is a TODO

Reply:
ok